README.md

```diff
@@ -24,3 +24,40 @@ Then:
     $ source ppenv/bin/activate
     $ pip install -r requirements.txt
     $ python setup.py develop
+
+## Creating an AWS Lambda Zipfile
+
+After modifying ``push_processor/settings.js`` for your appropriate settings, a
+Lambda-ready zipfile can be created with make:
+
+    $ make lambda-package
+
+Ensure that no extra files are present in the work directory, as all files in
```

Comment:
"All files in the work directory are included tin the lambda package, except any file listed in the ``ignore`` section of ``lambda.json``. Remove any other file from the work directory.